models/helper.go

```diff
@@ -25,6 +25,21 @@ func QueryFilterByConceptIdsHelper(query *gorm.DB, sourceId int, filterConceptId
 	return query
 }
 
+//
+func QueryFilterByConceptIdsHelper2(query *gorm.DB, sourceId int, filterConceptIds []int64,
+	omopDataSource *utils.DbAndSchema, resultSchemaName string, personIdFieldForObservationJoin string) *gorm.DB {
```

Comment:
copy of `QueryFilterByConceptIdsHelper`, with a different last param (`personIdFieldForObservationJoin`) just for testing now...